plugin-gestures/src/test/java/com/mapbox/maps/plugin/gestures/GesturesPluginTest.kt

```diff
@@ -640,6 +675,29 @@ class GesturesPluginTest {
     verify { cameraAnimationsPlugin.easeTo(any(), any()) }
   }
 
+  @Test
+  fun verifyRotateWithLowLevelAnimation() {
```

Comment:
let's rename test to identify that in this case `simultaneousRotateAndPinchToZoomEnabled = true`